srcpkgs/strongswan/template

```diff
@@ -7,9 +7,9 @@ build_style=gnu-configure
 configure_args="--disable-static --enable-blowfish --enable-curl
  --enable-eap-radius --enable-eap-mschapv2 --enable-eap-md5 --enable-led
  --enable-ha --enable-dhcp --enable-mediation --enable-soup --disable-des
- --enable-chapoly"
+ --enable-chapoly --enable-nm"
```

Comment:
You should bump the revision of the template so the package gets rebuilt with your changes.

Reply:
Of course